src/app/code/community/Shopgate/Cloudapi/sql/shopgate_cloudapi_setup/mysql4-upgrade-3.4.0-3.5.0.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/** @var Shopgate_Cloudapi_Model_Resource_Setup $installer */
+$installer = $this;
+$installer->startSetup();
+$connection = $installer->getConnection();
+$orderTable = $installer->getTable('shopgate_order_sources');
+if ($connection->tableColumnExists(
+        $orderTable,
+        Shopgate_Cloudapi_Model_Resource_Order_Source::COLUMN_AGENT
+    ) === false
+) {
+    $connection->addColumn(
+        $orderTable,
+        Shopgate_Cloudapi_Model_Resource_Order_Source::COLUMN_AGENT,
+        array(
+            'type'     => Varien_Db_Ddl_Table::TYPE_TEXT,
+            'nullable' => true,
+            'comment'  => 'User Agent',
+        )
+    );
+}
+$indexes = $connection->getIndexList($orderTable);
```

Comment:
I may need some help figuring out what the issue is that this script handles here, why are we expecting the index to possibly not exist here?